include/utility/json_utility.hpp

```diff
@@ -382,13 +446,13 @@ inline bool procesSetGpioTag(const nlohmann::json& i_parsedConfigJson,
             l_pinValue);
         return true;
     }
-    catch (const std::exception& ex)
+    catch (const std::exception& l_ex)
```

Comment:
Please avoid making changes which are not intended for the PR unless it is breaking the code. If some cosmetic change needs to go in, create an issue, club in that, once you have considerable amount there, go for the change.

Reply:
created issue - https://github.com/ibm-openbmc/openpower-vpd-parser/issues/376